pkgs/applications/networking/browsers/librewolf/src.json

```diff
@@ -1,11 +1,13 @@
 {
-  "packageVersion": "135.0-1",
+  "packageVersion": "135.0.1-1",
   "source": {
-    "rev": "135.0-1",
-    "hash": "sha256-UyKfgu6/VAfPkLCjlGgG8SGAG/nOyoP3uyesXqLc5Dk"
+    "rev": "135.0.1-1",
+    "hash": "sha256-UyKfgu6/VAfPkLCjlGgG8SGAG/nOyoP3uyesXqLc5Dk",
+    "sha256": "19w4734qbzhrsdhbzz9m82s7fhz1wndv0rkkbz6xjxwad94dgzy3"
```

Comment:
```suggestion
```
`hash` already exists. Setting `sha256` results in no rebuilds.

Reply:
Nice catch. I just ran the update script, and this explains why it is not updated aitomatically.

I won't be able to change it in the next 12 hours. Feel free to open another PR instead of this.